### ISSUE TITLE
Wrong border radius on BorderEffect shader is cut

### ISSUE THREAD

**Issue by albdima83** [lightning-js/renderer#297]
The view with border radius (50%) and border  width are cut.
See the picture attached:

<img width="376" alt="Screenshot 2024-06-06 alle 18 45 16" src="https://github.com/lightning-js/renderer/assets/62875440/1f9795b8-2263-4804-b27c-cb87b7b9545e">

Here a example code:

```javascript
const nodeCircleBorder = renderer.createNode({
    x: 100,
    y: 100,
    width: 152,
    height: 152,
    color: 0xff0000ff,
    shader: renderer.createShader('DynamicShader', {
      effects: [
        {
          type: 'radius',
          props: {
            radius: 152 / 2,
          },
        },
        {
          type: 'border',
          props: {
            width: 6,
            color: 0xfafafaff,
          },
        },
      ],
    }),
    parent: testRoot,
  });
```

**Comment by pecoram** [comment]
@jfboeve

**Comment by frank-weindel** [comment]
Fixed in #298
